x-pack/plugins/security_solution/cypress/tasks/exceptions.ts

```diff
@@ -69,10 +69,10 @@ export const editException = (updatedField: string, itemIndex = 0, fieldIndex =
 };
 
 export const addExceptionFlyoutItemName = (name: string) => {
-  cy.root()
-    .pipe(($el) => {
-      return $el.find(EXCEPTION_ITEM_NAME_INPUT);
-    })
+  cy.get(EXCEPTION_ITEM_NAME_INPUT).should('exist');
+  cy.get(EXCEPTION_ITEM_NAME_INPUT).should('be.visible');
+  cy.get(EXCEPTION_ITEM_NAME_INPUT, { timeout: 10000 }).first().focus();
```

Comment:
Looks good, just an fyi that @MadameSheema wants to move away from hardcoded timeouts if possible

Reply:
Thanks for the ping @michaelolo24!! and this is a good point to bring to my attention. The `	`defaultCommandTimeout` for Cypress executions on CI is, `150000`. If for any specific reason we want this to fail faster in case is not finding the element, it is ok to use it that way.  @dhurley14 can you please confirm if is this the case? Thanks!!

Reply:
Yup we could probably remove this - no need to override the default timeout. I think this was just leftover from when I was trying to fix some flake in the cypress test I wrote. Luckily, @MadameSheema fixed all that for me so I'll push an update to remove the timeout override. Thanks @michaelolo24 !!